circle.yml

```diff
@@ -17,3 +17,4 @@ test:
   # unfortunately these have to be two separate sbt commands ¯\_(ツ)_/¯
   - ./sbt coverage test && ./sbt coverageAggregate
   - ./sbt e2e:test assembly
+  - ./test-release.sh
```

Comment:
Thanks for adding this! How would you feel about calling it `test-config.sh` or something similar? That might be more appropriate since we'll now be running it on every commit (hooray!).